build.gradle

```diff
@@ -96,6 +100,12 @@ subprojects {
         externalJars (group: 'com.oracle.database.jdbc', name: 'ojdbc8', version: "${oracleDBDriverVersion}") {
             transitive = false
         }
+        externalJars (group: 'com.oracle.database.xml', name: 'xdb', version: "${oracleXDBVersion}") {
+            transitive = false
+        }
+        externalJars (group: 'com.oracle.database.xml', name: 'xmlparserv2', version: "${oracleXMLParserVersion}") {
```

Comment:
Let improve modul.md to mention all drivers that need be present for XML parsing.
Update the changelog

Reply:
updated the change log in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/10/commits/5c347d70db02692b3a10a12144d737eef71990de

will improve the module.md file in a separate PR with this linked issue. https://github.com/ballerina-platform/ballerina-standard-library/issues/1710